### ISSUE TITLE
Can you link to a FAQ on another site collection

### ISSUE THREAD

**Issue by sasrsc** [MastekApps/FaqPage#24]
We manage 4 site collections and we want to avoid duplication. Is there a way to link to a faq in another site collection?

**Comment by s-KaiNet** [comment]
No, it's not possible to link.

**Comment by MellowYellow85** [comment]
Is it possible to find the underlying list to use a workflow or powerautomate flow to sync lists on different sites?

Also, this is amazing. Lifesaver! Thank you!

**Comment by s-KaiNet** [comment]
Yes, simply go to FAQ Page start screen, then change the url to <your sharepoint site url>/Lists/Faq

**Comment by sasrsc** [comment]
That would be awesome…

Richard S. Clowes

**Comment by sasrsc** [comment]
I had a follow up question to this...I'm a citizen developer so please forgive the dumb question...but does the list live in our tenant (noted in the graphic as blurred out office365.com or that hash value afterwards. What does that hash value mean? Is it a re-direct to another tenant (ending in 7b8)?

![image](https://user-images.githubusercontent.com/8988793/92919790-1ef7fc80-f3ff-11ea-95e9-0bff1fe611d5.png)

**Comment by sasrsc** [comment]
I have tried @MellowYellow85 suggestion of the Power Automate routine but I don't see the "lists"

I can see all my FAQ sets (as lists) here by editing the URL like you suggested...
![image](https://user-images.githubusercontent.com/8988793/92920857-f83ac580-f400-11ea-9098-983d1bd7453c.png)

But when I try a Power Automate, it trims the custom value ... as I paste in the value above it removes the /Lists/Faq and leaves me with FAQPage and all it finds are a "documents" library and not any of the lists.

![image](https://user-images.githubusercontent.com/8988793/92920940-20c2bf80-f401-11ea-86f3-534762300b3b.png)

I'm sure it's something very simple and maybe it has to do with the tenant name from ournameSASOffice365 to ournameSASOffice365-hash value?

**Comment by s-KaiNet** [comment]
In the site address field try to provide your real tenant address. 
For example, on your screen, you have `https://organization-Office365-243b434j234.sharepoint.com/sites/travel/etc..`. Change the left part to the root url of your tenant, so it should look like  `https://organization-Office365.sharepoint.com/sites/travel/etc..` (without these ugly `-243b434j234` part).

**Comment by sasrsc** [comment]
When I do that... I can only get as far as adding /FAQPage. 
If I add anything beyond that such as /FAQPage/Lists/Faq it will trim it ...

![image](https://user-images.githubusercontent.com/8988793/92921676-81063100-f402-11ea-9d24-ec0b45c771a5.png)

**Comment by s-KaiNet** [comment]
This is how it looks like for me. Maybe you just need GetItms action?  

![image](https://user-images.githubusercontent.com/10658276/92926437-ac117480-f444-11ea-8734-fdf3588f8baf.png)

**Comment by sasrsc** [comment]
I was doing a stupid user thing.
I was assuming I would have to state the specific FAQ set name.
You don’t. Of course the response is very long but at least this gives me access to the lists and all the questions.

![image](https://user-images.githubusercontent.com/8988793/92993812-ac505500-f4c2-11ea-8818-efff97daee6a.png)

![image](https://user-images.githubusercontent.com/8988793/92993807-a490b080-f4c2-11ea-926f-32f6907c6398.png)


What I’m not clear about is how I would add an item to an existing FAQ set through the standard Power Automate action item.
Each of these questions is associated with a FAQ set which in Sharepoint is a folder.
How would I specifically state this needs to be in this FAQ sets vs. a different FAQ set?
It doesn’t seem like the Content Type id changes???

"{Path}": "Lists/Faq/Credit Card Questions/",
        "{FullPath}": "Lists/Faq/Credit Card Questions/33_.000",
        "{ContentType}": {
          "@odata.type": "#Microsoft.Azure.Connectors.SharePoint.SPListExpandedContentType",
          "Id": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx4608165C",
          "Name": "Item"
        },


[cid:image007.png@01D688CD.24EDB640]


Richard S. Clowes
Senior Director | Travel Operations | Corporate Services Division​​
Tel: 919-531-7955 (work) | 919-413-3084 (cell) | [cid:image002.png@01D688CB.1886E2C0] <https://teams.microsoft.com/l/chat/0/0?users=Richard.Clowes@sas.com>
Email: Richard.Clowes@sas.com
SAS Institute Inc. | 100 SAS Campus Drive | Cary, NC 27513​​
www.sas.com<http://www.sas.com/>
​​[cid:image003.jpg@01D688CB.1886E2C0]
​
